### DISCUSSION TITLE
Scripting on ESP8266: ssize and SCRIPT_MAXSSIZE 128 are not effective

### DISCUSSION THREAD

**Discussion by jkwim** [arendst/Tasmota#21847] [Q&A] [answered]
I have compiled SCRIPTING feature from latest Development branch `[Tasmota 14.1.0.3 (scripting-btnst-udspl) by Theo Arends]` with following options
```
        #ifndef USE_SCRIPT
          #define USE_SCRIPT
        #endif
		
        // Serial Bridge
        #define USE_SERIAL_BRIDGE
 
          // for serial IO support
        #define USE_SCRIPT_SERIAL

        #define MAXSVARS 4
        // Max Variable size in SCRIPTS
        #define SCRIPT_MAXSSIZE 128 <=================
 
        #define USE_SCRIPT_WEB_DISPLAY
        #define USE_SCRIPT_JSON_EXPORT
		
        #define USE_BUTTON_EVENT

        #define USE_SCRIPT_TIMER		

```        
Have increased the size to 24 in >D section

```
>D
ssize=24

```
However string variables get restricted to default 19 characters.

```
tstr="123456789012345678901234"
dt [Ci5f1s1x2y226p24]%tstr%

```

Output will be
 
`1234567890123456789`

Have I done something wrong?

**Comment by sfromis** [comment, marked as answer]
Not using scripting, but the [docs](https://tasmota.github.io/docs/Scripting-Language/#script-sections) suggests that the format would be simply `>D 24`
(The next line is just an explanation of what `ssize` means)

**Reply by jkwim** [reply]
Ah! Ha! That did the trick!

Thanks a lot for the quick reply.
